src/api/types/contact.ts

```diff
@@ -0,0 +1,13 @@
+export const ContactAwaitingAcceptance = 'awaiting-acceptance'
+export const ContactNotConfirmed = 'not-confirmed'
+export const ContactConfirmed = 'confirmed'
+
+export type ContactStatus = typeof ContactAwaitingAcceptance | typeof ContactNotConfirmed | typeof ContactConfirmed
+
+export type Contact = {
```

Comment:
These field names, most likely, don't align with what is implemented in the backend.

Reply:
I'm going to update this type in the next PR, during implementation of actual server requests. 
Is it OK for you @dorzepowski ?